examples/watch_resources/README.md

```diff
@@ -0,0 +1,50 @@
+# Watching for Resource Changes
+
+The test harness supports several methods for querying Kubernetes object types and watching for resource states. This example shows how to watch particular resource and how to register the functions to act upon based on the events recieved.
+
+
+# Watch for the deployment and triger action based on the event
+
+Watch has to run as goroutine to get the different events based on the k8s resource state changes.
+```go
+func TestWatchForResources(t *testing.T) {
+...
+dep := appsv1.Deployment{
+				ObjectMeta: metav1.ObjectMeta{Name: "watch-dep", Namespace: cfg.Namespace()},
+			}
+
+			// watch for the deployment and triger action based on the event recieved.
+			go cl.Resources().Watch(ctx, &appsv1.DeploymentList{}, &client.ListOptions{
+				FieldSelector: fields.OneTermEqualSelector("metadata.name", dep.Name),
+				Namespace:     dep.Namespace}, cl.RESTConfig()).SetAddFunc(onAdd).RegisterDeleteFunc(onDelete).Start()
```

Comment:
#nit,  I think you missed updating this to `SetDeleteFunc` from `RegisterDeleteFunc`

Reply:
```suggestion
				Namespace:     dep.Namespace}, cl.RESTConfig()).SetAddFunc(onAdd). SetDeleteFunc(onDelete).Start()
```